encoding/protobq/schema.go

```diff
@@ -80,6 +83,27 @@ func (o SchemaOptions) inferFieldSchema(field protoreflect.FieldDescriptor) *big
 	return fieldSchema
 }
 
+func appendNonEmpty(parts []string, value string) []string {
+	trimmed := strings.TrimSpace(value)
+	if len(trimmed) > 0 {
+		parts = append(parts, trimmed)
+	}
+	return parts
+}
+
+func (o SchemaOptions) buildDescription(field protoreflect.FieldDescriptor) string {
+	if !o.UseProtoCommentsAsDescription {
+		return ""
+	}
+	sourceLocation := field.ParentFile().SourceLocations().ByDescriptor(field)
+	var parts []string
+	for _, comment := range strings.Split(sourceLocation.LeadingComments, "\n") {
+		parts = appendNonEmpty(parts, comment)
+	}
+	parts = appendNonEmpty(parts, sourceLocation.TrailingComments)
+	return strings.Join(parts, "\n")
+}
+
 func (o SchemaOptions) inferOneofFieldSchema(oneof protoreflect.OneofDescriptor) *bigquery.FieldSchema {
 	fieldSchema := &bigquery.FieldSchema{
```

Comment:
No description for oneofs?

Reply:
I realise now that it might not be applicable...

Reply:
I think it might be possible, I'll make a try at it.